packages/reports/src/Containers/statement.tsx

```diff
@@ -21,8 +21,8 @@ import { ReportsMeta } from '../Components/reports-meta';
 import EmptyTradeHistoryMessage from '../Components/empty-trade-history-message';
 import { observer, useStore } from '@deriv/stores';
 import { useReportsStore } from 'Stores/useReportsStores';
-import { TSupportedContractType, TUnsupportedContractType } from 'Types';
-import { TSource } from '@deriv/components/src/components/data-table/data-table';
+import { TUnsupportedContractType } from 'Types';
+import { TSource } from '@deriv/components/src/components/data-table/table-row';
```

Comment:
There was an error that `TSource` is not in _data-table.tsx_. As I checked it was in _table-row.tsx_